src/Requests/CollectionTtl.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+namespace Momento\Requests;
+
+class CollectionTtl
+{
+    private ?int $ttlSeconds;
+    private ?bool $refreshTtl;
+
+    public function __construct(?int $ttlSeconds = null, ?bool $refreshTtl = true)
+    {
+        $this->ttlSeconds = $ttlSeconds;
+        $this->refreshTtl = $refreshTtl;
+    }
+
+    public static function fromCacheTtl(): CollectionTtl
+    {
+        return new CollectionTtl(null, true);
+    }
+
+    public static function of(int $ttl): CollectionTtl
```

Comment:
I think I'd prefer this parameter to match the constructor and be called `$ttlSeconds` instead, so the unit is explicit in the parameter name (and in the named parameter label). I know I personally get confused about when we're using seconds vs. millis.